app/models/health_care_application.rb

```diff
@@ -287,13 +287,13 @@ def form_matches_schema
   end
 
   def current_schema
-    schema = VetsJsonSchema::SCHEMAS[self.class::FORM_ID]
-    return schema unless Flipper.enabled?(:hca_use_facilities_API)
-
     Rails.logger.warn(
-      "HealthCareApplication::hca_use_facilitiesAPI enabled = #{Flipper.enabled?(:hca_use_facilities_API)}"
+      "HealthCareApplication::hca_use_facilitiesAPI enabled = #{Flipper.enabled?(:hca_use_facilities_API, user)}"
```

Comment:
We could move `Flipper.enabled?(:hca_use_facilities_API, user)` to a variable and use it for the log and conditional check. That way we don't accidentally get out of sync.

Reply:
good point - I wasn't considering it given that this ought to be short-lived.

Reply:
Yeah it's probably not a big deal either way, I just figured it would lower the risk of them being out of sync 🤷 . I'm fine either way.